core/src/main/java/de/danoeh/antennapod/core/service/playback/LocalPSMP.java

```diff
@@ -663,8 +671,18 @@ public void setVolume(final float volumeLeft, float volumeRight) {
      */
     private void setVolumeSync(float volumeLeft, float volumeRight) {
         playerLock.lock();
-        mediaPlayer.setVolume(volumeLeft, volumeRight);
-        Log.d(TAG, "Media player volume was set to " + volumeLeft + " " + volumeRight);
+        if (media != null && media.getMediaType() == MediaType.AUDIO) {
```

Comment:
What about video feeds? They now just keep the previous volume.

Reply:
It should apply to video as well, yes. I shall amend.